homebrew_challenge/src/pages/AboutUs.jsx

```diff
@@ -94,6 +98,46 @@ const AboutUs = () => {
       );
     });
 
+  const vendors = [
+    {
+      name: "Atlassian",
+      vendorurl: "https://www.atlassian.com/",
+      logoref: "../assets/vendorLogos/atlassian.svg",
+    },
+    {
+      name: "AWS",
+      vendorurl: "https://aws.amazon.com/",
+      logoref: "../assets/vendorLogos/aws.svg",
+    },
+    {
+      name: "Slack",
+      vendorurl: "https://slack.com/",
+      logoref: "../assets/vendorLogos/slack.svg",
+    },
+    {
+      name: "Stadia Maps",
+      vendorurl: "https://stadiamaps.com/",
+      logoref: "../assets/vendorLogos/stadiamaps.svg",
+    },
+  ];
+
+  const sortedVendors = vendors
+    .sort((a, b) => a.name.localeCompare(b.name))
+    .map((data, index) => {
+      return (
```

Comment:
Would be worth splitting the common code with line 84 onwards into helper functions rather than duplicate the code